service/initialise.go

```diff
@@ -65,6 +68,17 @@ func (e *ExternalServiceList) GetKafkaConsumer(ctx context.Context, cfg *config.
 	return kafkaConsumer, nil
 }
 
+// GetS3Clients returns S3 clients public and private. They share the same AWS session.
+func (e *ExternalServiceList) GetS3Clients(cfg *config.Config) (s3Public S3Client, s3Private S3Client, err error) {
+	s3Public, err = e.Init.DoGetS3Client(cfg.AwsRegion, cfg.PublicBucketName, true, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	s3Private, _ = e.Init.DoGetS3Client("", cfg.PrivateBucketName, true, s3Public.Session())
```

Comment:
I think it is possible for this to still return a valid error, but I may be missing something? If so I think it would likely be worth returning it so that we can ensure that we can log and identify the error.

Reply:
I ignored the error because if we reuse the session, the client creation does not return any error. But maybe at some point we want to change something in DoGetS3Client, so I will add the error checking here.